internals/overlord/logstate/loki/loki.go

```diff
@@ -169,16 +178,27 @@ func (c *Client) buildRequest() lokiRequest {
 	for _, service := range services {
 		entries := bucketedEntries[service]
 		stream := lokiStream{
-			Labels: map[string]string{
-				"pebble_service": service,
-			},
+			Labels:  c.getLabels(service),
 			Entries: entries,
 		}
 		req.Streams = append(req.Streams, stream)
 	}
 	return req
 }
 
+func (c *Client) getLabels(serviceName string) map[string]string {
```

Comment:
Instead of allocating and building this map every `Flush`, can we only do it every `SetLabels` call? It shouldn't be any harder, and it would make the implementation more efficient.